packages/internal/dex/sdk/src/types/index.ts

```diff
@@ -97,6 +97,16 @@ export type TransactionResponse = {
   quote: Quote;
 };
 
+/**
+ * Type representing the details of a wrap or unwrap transaction
+ * @property {@link TransactionDetails} transaction - The wrap or unwrap transaction
+ * @property {@link TransactionDetails | null} approval - The approval transaction or null if it is not required
+ */
+export type WrapUnwrapTransactionDetails = {
+  transaction: TransactionDetails,
+  approval: TransactionDetails | null,
```

Comment:
Does not providing a quote mean Checkout will need to handle this case differently on their side?

Reply:
I think so - we should be able to provide a Quote object still, and IMO a Quote still makes sense, even if it is fixed.

Without these details I don't think the consumer of this SDK can provide a breakdown of the transaction prior to the user initiating

Reply:
In the context of a wrap/unwrap, it could just look like:

```typescript
const wrapQuote = {
  amount: 100,
  amountWithMaxSlippage: 100,
  slippage: 0, // No slippage gets applied
  fees: [] // No fees get applied
}
```

Reply:
agree

Reply:
We still provide a quote at the top level. This type was made for the internal details where we split off for wrapping/unwrapping.

Reply:
@pano-skylakis yep that's what we are doing already